static/js/humhub/humhub.ui.codemirror.js

```diff
@@ -8,21 +8,34 @@ humhub.module('ui.codemirror', function(module, require, $) {
 
     var init = function () {
         event.on('humhub:ready', function (evt) {
```

Comment:
@s-tyshchenko Did you tried this event here?
https://github.com/humhub/humhub/blob/master/static/js/humhub/humhub.client.pjax.js#L44